sdk/monitor/monitor-opentelemetry/samples-dev/src/01-basic-connection.ts

```diff
@@ -0,0 +1,60 @@
+/**
```

Comment:
This file and `02-environment-connection` appear to be very similar as they both demonstrate connecting to an app insights resource using an environment variable.